src/glyph/mod.rs

```diff
@@ -220,6 +233,194 @@ pub struct AffineTransform {
     pub y_offset: f32,
 }
 
+impl Anchor {
+    pub fn new(
+        x: f32,
+        y: f32,
+        name: Option<String>,
+        color: Option<Color>,
+        identifier: Option<Identifier>,
+        lib: Option<Plist>,
+    ) -> Self {
+        Self { x, y, name, color, identifier, lib }
+    }
+
+    /// Returns an immutable reference to the anchor's lib.
+    pub fn get_lib(&self) -> &Option<Plist> {
```

Comment:
No need for `get_` in getters; you can just call this `lib`.

Also: we should return `Option<&Plist>` instead of `&Option<Plist>`; you can do this with `self.lib.as_ref()`.

Reply:
This seems to mess with innocuous looking code:

```
   --> src/glyph/builder.rs:155:50
    |
155 |         insert_identifier(&mut self.identifiers, guideline.identifier().clone())?;
    |                                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `shared_types::Identifier`, found `&shared_types::Identifier`
    |
    = note: expected enum `std::option::Option<shared_types::Identifier>`
               found enum `std::option::Option<&shared_types::Identifier>`
```

Reply:
Ah. I have to use `cloned()`.